### PR TITLE
fix etcd prefix

### PR DESCRIPTION
**Pull request by AmoebaProtozoa**
Signed-off-by: David <8039876+AmoebaProtozoa@users.noreply.github.com>

### CODE REVIEW

File: handle/enabledKeyspaceConnect.go [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+		arr := strings.Split(line, ",")
+		if len(arr) < 1 {
+			log.Error("failed to get keyspace name from dumped file")
+			continue
+		}
+		keyspaceName = arr[1]
+		if err = connectToKeyspace(ctx, keyspaceName, endpoint, ca); err != nil {
+			log.Info(err.Error())
+		}
+		time.Sleep(1 * time.Second)
```
Comment (ystaticy) [inline handle/enabledKeyspaceConnect.go]: We need to make the sleep time settable

File: main.go [hunk 2]
```diff
@@ -156,6 +169,26 @@ func main() {
 			handle.LoadRegionLablesAndGC(dumpFileRegionLabelRule, dumpfilePath, ctx, []string{*pdAddr}, isCanRun, *isSkipConfirm)
 
 		}
+	case opDumpEnabledKS:
+		{
+			dumpfile, err := os.OpenFile(*dumpFilepath, os.O_CREATE|os.O_WRONLY|os.O_TRUNC, 0666)
```
Comment (ystaticy) [inline main.go]: dumpFilepath is used to store archived keyspace list, so we can set another dump file path to store the enabled keyspace

File: handle/reformatEtcdPath.go [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+		return nil
+	}
+
+	// Create mutex for concurrency control.
+	session, err := concurrency.NewSession(cli, concurrency.WithContext(ctx))
+	if err != nil {
+		return err
+	}
+	defer session.Close()
+	mu = concurrency.NewMutex(session, reformatLock)
```
Comment (iosmanthus) [inline handle/reformatEtcdPath.go]: Why should we use an `etcd` lock here, seems multiple reformat tasks are allowed to run simultaneously.

Comment (AmoebaProtozoa) [reply handle/reformatEtcdPath.go]: It's here to prevent multiple instance of the tool to run.
But it's probably fine to remove it.

**Comment by ystaticy** [comment]
LGTM

**Review by iosmanthus — APPROVED** [review]

**Pull request closed without merging**